backend/src/main/server.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"backend"
+	"backend/routes"
+	_ "github.com/go-sql-driver/mysql"
+	"net/http"
+)
+
+func main() {
+	if backend.Config == nil {
+		panic("Could not find configuration file, shutting down")
+	}
+
+	println("Server has started on port 8080")
```

Comment:
https://golang.org/pkg/builtin/#println

I'm going to change this

Reply:
yeah go ahead!